pkgs/by-name/es/espflash/package.nix

```diff
@@ -38,7 +38,7 @@ rustPlatform.buildRustPackage rec {
     SystemConfiguration
```

Comment:
Some frameworks may be missing on aarch64-darwin. Is the package known to build on Darwin before?

Reply:
I'm not a Darwin user so I cannot say.